packages/mimecast/data_stream/audit_events/elasticsearch/ingest_pipeline/default.yml

```
@@ -55,55 +63,100 @@ processors:
+      pattern: "%{mimecast.info}, %{event.type}, %{mimecast.search}, %{mimecast.rest_of_event_info}"
       if: 'ctx?.event?.action == "remediation-incident-adjustment"'
   - dissect:
       field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{mimecast.type}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}"
+      pattern: "%{mimecast.info}, %{mimecast.type}, %{mimecast.rest_of_event_info}"
       if: 'ctx?.event?.action == "review-set-action"'
   - dissect:
       field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}"
-      if: 'ctx?.event?.action == "archive-mailbox-export-download" || ctx?.event?.action == "archive-mailbox-restore"' #logon-authentication-failed
-  - dissect:
-      field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{mimecast.description}, %{mimecast.product}, %{mimecast.provider}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}"
+      pattern: "%{mimecast.info}, %{mimecast.description}, %{mimecast.product}, %{mimecast.provider}, %{mimecast.rest_of_event_info}"
       if: 'ctx?.event?.action == "connectors-management"'
   - dissect:
       field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{mimecast.criteria}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}"
-      if: 'ctx?.event?.action == "search-action"'
-  - dissect:
-      field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{?key}: <%{mimecast.name.from}> %{email.from.address}, %{?key}: <%{mimecast.name.to}> %{email.to.address}, %{?key}: %{email.subject}, %{?key}: %{email.origination_timestamp}, %{?key}: %{mimecast.viewed}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}"
-      if: 'ctx?.event?.action == "message-action"'
-  - dissect:
-      field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{?key}: %{mimecast.date}, %{?key}: %{mimecast.time}, %{?key}: %{client.ip}, %{?key}: %{mimecast.application}, %{mimecast.application_method}, %{event.reason}"
+      pattern: "%{mimecast.info}, %{?key}:%{mimecast.email.address}[%{mimecast.email.metadata}] %{?key}: %{client.ip} %{?key}: %{mimecast.application}"
       if: 'ctx?.event?.action=="logon-authentication-failed"'
       ignore_missing: true
       ignore_failure: true
   - dissect:
       field: mimecast.eventInfo
-      pattern: "%{mimecast.info}, %{?key}:%{mimecast.email.address}[%{mimecast.email.metadata}] %{?key}: %{client.ip} %{?key}: %{mimecast.application}"
-      if: 'ctx?.event?.action=="logon-authentication-failed"'
+      pattern: "%{mimecast.info}, %{mimecast.rest_of_event_info}"
```

```suggestion
      pattern: "%{mimecast.info}, %{mimecast.rest_of_event_info}"
      if: 'ctx.mimecast?.info == null'
```

Without this condition this dissect will get executed for every event, so we either add it or if this works, we could remove all the previous, more specific dissect processors.

---

condition added